### PR TITLE
MWPW-141206 AA University in Marketo

### PR DESCRIPTION
**Pull request by meganthecoder**
* Enable AA University registration for Marketo (by metadata)

Resolves: [MWPW-141206](https://jira.corp.adobe.com/browse/MWPW-141206)

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.page/drafts/methomas/marketo-aa-university
- After: https://methomas-aa-sandbox--bacom--adobecom.hlx.page/drafts/methomas/marketo-aa-university

### CODE REVIEW

**Comment by Brandon32** [comment]
Look good, if things get more complicated we might want to look into using the `milo:deferred` event.

**Comment by meganthecoder** [comment]
I removed the scripts test due to time constraints and pulled in into a new ticket https://jira.corp.adobe.com/browse/MWPW-141665

**Review by Brandon32 — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

File: scripts/aa-university.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    method: 'POST',
+    body: JSON.stringify({
+      first_name: firstName?.value,
+      last_name: lastName?.value,
+      email: email?.value,
+      university: 'none',
+      country: country?.value,
+    }),
+  })
+    .catch((error) => window.lana.log(`Marketo AA University Error: ${error}`));
```
Comment (overmyheadandbody) [inline scripts/aa-university.js:17]: Any chance `window.lana` might not exist at this point, justifying the use of some optional chaining?

**Review by overmyheadandbody — APPROVED** [review]

File: scripts/aa-university.js (hunk 1 above)
Comment (meganthecoder) [reply scripts/aa-university.js:17]: That's a good thought, but I don't think so. This is being run after loadArea and we don't even use optional chaining there within block code

**Pull request merged**